modules/tests/test_ref_modules/test_ref_modules.cpp

```diff
@@ -546,7 +546,7 @@ TEST_F(RefModulesTest, UpdateDevicePower)
     device1Channel.release();
     device0Signal.release();
     device1Signal.release();
-//    rendererFb.release();
+    //    rendererFb.release();
```

Comment:
These indentation changes seem somewhat odd. If changed in this manner, there should not be a whole tab following the "//", but a single space.